approved_tags.yaml

```diff
@@ -0,0 +1,103 @@
+---
+- "accessibilità"
+- "ia"
+- "android"
+- "api"
+- "apl"
+- "arte"
+- "domanda"
+- "assembly"
+- "audio"
+- "libro"
+- "browsers"
+- "c"
+- "c++"
+- "clojure"
+- "scienzecognitive"
+- "compilatori"
+- "informaticateorica"
+- "crittografia"
+- "criptovalute"
+- "css"
+- "cultura"
+- "d"
+- "database"
+- "debugging"
+- "design"
+- "devops"
+- "sistemidistribuiti"
+- "dotnet"
+- "dragonflybsd"
+- "istruzione"
+- "elixir"
+- "elm"
+- "emacs"
+- "erlang"
+- "evento"
+- "metodiformali"
+- "fortran"
+- "freebsd"
+- "giochi"
+- "go"
+- "grafica"
+- "hardware"
+- "haskell"
+- "storia"
+- "illumos"
+- "ios"
+- "java"
+- "javascript"
+- "lavoro"
+- "legge"
+- "linux"
+- "lisp"
+- "lua"
+- "mac"
+- "matematica"
+- "meta"
+- "ml"
+- "mobile"
+- "netbsd"
+- "reti"
+- "networking"
+- "nodejs"
```

Comment:
secondo me basta javascript, è troppo specifico

Reply:
Io lo lascerei

Reply:
JavaScript magari viene usato per parlare del linguaggio in sé mentre NodeJS si riferisce a tutto l'ecosistema. Io lo lascerei.

Reply:
però se teniamo nodejs dovremmo tirare in ballo una marea di ecosistemi molto più importanti e diffusi. Imho non ne vale la pena.